### PR TITLE
Job to enable pr-check retest on backstage-showcase

### PR DESCRIPTION
**Pull request by subhashkhileri**
**Job to enable pr-check retest on backstage-showcase** 

enable retriggering PR test without making additional no-op commit 
https://issues.redhat.com/browse/RHIDP-1687

### CODE REVIEW

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-images

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-images

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-showcase-e2e-main-v4.13-backstage-showcase-pr-check

File: ci-operator/config/janus-idp/backstage-showcase/OWNERS [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/janus-idp/showcase-e2e root OWNERS
```
Comment (kadel) [inline ci-operator/config/janus-idp/backstage-showcase/OWNERS:2]: I thought that we are not using showcase-e2e anymore

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-v4.13-backstage-showcase-pr-check

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-e2e-tests

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-e2e-tests

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-e2e-tests

**Comment by subhashkhileri** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-e2e-tests

**Review by josephca — APPROVED** [review]
/lgtm
/approve

**Pull request merged**